spec/unit/defines_spec.rb

```diff
@@ -35,6 +35,7 @@
         FLAnimatedImage
         FLWeakProxy
         FLAnimatedImageView
+        BrianSwiftAsyncIntegration
```

Comment:
Can we add a test for SwiftUI mangling since you made a change to include it?

Reply:
Yep. I've added `SwiftUI` to the symbols text file. The idea is that this _isn't_ included in the mangling. So, we don't want to add an additional check here.

Reply:
Gotcha. Thanks!